<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionalDlgClass</class>
 <widget class="QWidget" name="ConditionalDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConditionalDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>  比较算子LT_Z：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>  比较算子GT_Y：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>  比较算子LT_Y：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>  比较算子GT_X：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>  比较算子LT_X：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LT_X">
       <property name="text">
        <string>3.0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>  保持点云结构：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="KeepOrganized">
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>  比较算子GT_Z：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="GT_X">
       <property name="text">
        <string>-3.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="GT_Y">
       <property name="text">
        <string>-3.0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="LT_Y">
       <property name="text">
        <string>3.0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="GT_Z">
       <property name="text">
        <string>-3.0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="LT_Z">
       <property name="text">
        <string>3.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
